common/lv_conf.h

```diff
@@ -223,7 +223,7 @@ typedef void * lv_indev_drv_user_data_t;            /*Type of user data in the i
  *===============*/
 
 /*1: Enable the log module*/
-#define LV_USE_LOG      0
+#define LV_USE_LOG      1
```

Comment:
this bumps the firmware binary size by `3256 bytes`. we might want to do this by different mechanism (directly modifying the `lv_mem_assert` macro in `lv_mem.h`.